src/hooks/use-home-page.hook.ts

```diff
@@ -30,8 +35,14 @@ function useHomePage(): HomePageHook {
   const email = localStorage.getItem('email')
   const avatarText = userInfo.name[0] + userInfo.surname[0]
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
-  const [selectedMenu, setSelectedMenu] = useState<string>(`${t('t-profile')}`)
+  const location = useLocation()
+  const queryParams = new URLSearchParams(location.search)
+  const [selectedMenu, setSelectedMenu] = useState<string>(t('t-profile'))
   const open = Boolean(anchorEl)
+  const isTrackingAndTablet = userInfo.notices
+  const userBMI = Math.floor(userInfo.weight / (userInfo.height / 100) ** 2)
```

Comment:
could you avoid of using magic numbers, let's create constants for this numbers

Reply:
i don't understand, that you mean? BMI calculates based on user data

Reply:
in this case magic numbers are 100 and 2